go-controller/pkg/ovn/namespace.go

```diff
@@ -131,6 +131,16 @@ func (oc *Controller) multicastDeleteNamespace(ns *kapi.Namespace, nsInfo *names
 	nsInfo.multicastEnabled = false
 }
 
+func updateHybridOverlayAnnotations(nsInfo *namespaceInfo, ns *kapi.Namespace) {
+	if annotation := ns.Annotations[hotypes.HybridOverlayExternalGw]; annotation != "" {
+		if ip := net.ParseIP(annotation); ip != nil {
+			nsInfo.hasHybridOverlayExternalGW = true
```

Comment:
while you're doing random cleanups you should make the capitalization of "gw" consistent throughout. eg `hotypes.HybridOverlayExternalGw` vs `nsInfo.hasHybridOverlayExternalGW`